src/CMakeLists.txt

```
@@ -184,7 +184,8 @@ target_link_libraries (alivedetection-test openvas_nasl_shared cgreen
                        ${LIBGVM_BASE_LDFLAGS}
                        ${GLIB_LDFLAGS}
                        ${PCAP_LDFLAGS}
-                       ${LINKER_HARDENING_FLAGS} ${CMAKE_THREAD_LIBS_INIT})
+                       ${LINKER_HARDENING_FLAGS} ${CMAKE_THREAD_LIBS_INIT} 
+                       -Wl,-wrap,socket,-wrap,setsockopt)
```

Instead of using the flags directly here, it would be nice to have something like
```
set (LINKER_WRAP_OPTIONS     "-Wl,-wrap,socket,-wrap,setsockopt")
```
and then 
```suggestion
                       ${LINKER_WRAP_OPTIONS})
```